config/install/system.site.yml

```diff
@@ -0,0 +1,10 @@
+name: ezcontent
```

Comment:
@arshadkhan53 why do we need this file ?

Reply:
only if we set default front page as login page, we don't require it otherwise, I have removed it.